torch/csrc/cuda/CUDAPluggableAllocator.cpp

```diff
@@ -245,6 +257,20 @@ void CUDAPluggableAllocator::endAllocateStreamToPool(
   }
 }
 
+void CUDAPluggableAllocator::incOngoingCaptures(
+    int device) {
+  if (inc_ongoing_captures_fn_) {
```

Comment:
TODO: add warnings here as attempting to call `incOngoingCaptures` and having it be a no-op is potentially fatal as downstream recording of events will be incorrect.